cytotable/sources.py

```diff
@@ -45,64 +46,100 @@ def _build_path(
 def _get_source_filepaths(
     path: Union[pathlib.Path, AnyPath],
     targets: List[str],
-    source_datatype: Optional[str] = None,
 ) -> Dict[str, List[Dict[str, Any]]]:
     """
     Gather dataset of filepaths from a provided directory path.
 
     Args:
         path: Union[pathlib.Path, Any]:
-            Path to seek filepaths within.
+            Path as directory to seek filepaths within or file reference.
```

Comment:
```suggestion
            Either a directory path to seek filepaths within or a path directly to a file.
```

Consider also specifying something to the extend that the Path can be local or on the cloud.

Reply:
suggestion just to boost documentation clarity

Reply:
So much better, thanks!